etc/disable-common.inc

```diff
@@ -172,3 +172,29 @@ blacklist ${PATH}/roxterm-config
 blacklist ${PATH}/terminix
 blacklist ${PATH}/urxvtc
 blacklist ${PATH}/urxvtcd
+
+# disable common suid programms
+blacklist ${PATH}/firejail
```

Comment:
@manevich as you said, wouldn't `nonewprivs` and `noroot` already disable most/all of this behavior?